### ISSUE TITLE
Play buttons in "Saved text" area should load audio from server?

### ISSUE THREAD

**Issue by HannaLindgren** [stts-se/chromedictator#54]
At least when it's under development, this could be a neat way to find errors, or rather, to verify the server audio from the browser.

**Comment by jensedlund** [comment]
Agreed. Perhaps a separate "Play from server" button? Or a switch somewhere.
